src/wizard/wizard.scss

```diff
@@ -4,47 +4,42 @@
 
 @mixin iui-wizards {
   @include iui-reset;
-
   @include iui-font-family;
 
   > .iui-wizards-wrapper {
     display: flex;
     flex-direction: row;
     justify-content: space-around;
+    align-items: flex-start;
```

Comment:
Ah, just noticed this one (completely missed when there were tons of changes 😄 )

I was wondering how the alignment issues were magically fixed